rosa_architecture/rosa-sts-getting-started-workflow.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="rosa-sts-understanding-the-deployment-workflow"]
```

Comment:
This has been an issue throughout the ROSA docs. I might come back to this one in this PR, but it might be better to open a follow-up to make it easier to update xrefs.